README.md

```diff
@@ -57,11 +57,52 @@ Check out our [issues](https://github.com/ACM-VIT/Fill-In-the-Blanks/issues) and
 ---
 ## Overview
 
-The overview starts here. Random text about the project, motive, how, what, why etc.
+### Idea
+Fill in the Blanks, but with Images!
+
+### Explanation
+The aim is to build a deep learning model, that takes as input an image with a missing rectangular portion and a boolean mask indicating its location, and imagines the missing content. The basic set of packages can be found in requirements.txt and can be installed using the pip command from usage section. The suggested dataset consists of images of various indoor scenes. Use the provided code to create the blanks in the images.
+Packages used are: TensorFlow for creating and training the model, NumPy for handling arrays, MatPlotLib for image output.
```

Comment:
Instead of mentioning packages, create a requirements.txt file with all the needed packages

And then you could add in the readme

## Usage
`pip install -r requirements.txt`